app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Wrong copy command input")
+    cmd = command.split()[0]
+    data_file = command.split()[1]
+    data_file_copy = command.split()[2]
+    if cmd == "cp" and data_file_copy != data_file_copy:
```

Comment:
The condition 'data_file_copy != data_file_copy' is always false because it compares the variable to itself. This should likely be 'data_file != data_file_copy' to ensure the source and destination files are not the same.